docs/getting-started/installation.md

```diff
@@ -4,9 +4,21 @@ Decompose provides a number of modules, they are all published to Maven Central
 
 ## The main Decompose module
 
-The main functionality is provided by the `decompose` module. It contains some core features like [ComponentContext](../component/overview.md#componentcontext), [Child Stack](../navigation/stack/overview.md), etc.
+The main functionality is provided by the `decompose` module. It contains some core features like:
 
-### Gradle setup
+- [ComponentContext](../component/overview.md#componentcontext) needed to allow custom components
+to manage lifecycle, state saving,...
+- [Value](../component/overview.md/#value-and-mutablevalue-state-holders) allowing UI code to observe value changes
```

Comment:
```suggestion
- [Value](../component/overview.md/#value-and-mutablevalue-state-holders) - Swift-friendly observable state holder (allows observing state changes in the UI).
```

Reply:
Can we somehow make it more clear that this is the class that is used within both Android and iOS/Swift? 
For example 
"Multiplatform (Swift friendly) observable state holder (allows observing state changes in the UI)."

Reply:
Sure! Updated the suggestion.